module/solutions/helper.js

```diff
@@ -2271,7 +2268,10 @@ module.exports = class SolutionsHelper {
 					{
 						$facet: {
 							totalCount: [{ $count: 'count' }],
-							data: [{ $skip: pageSize * (pageNo - 1) }, { $limit: pageSize }],
+							data: [
+								{ $skip: CONSTANTS.common.DEFAULT_PAGE_SIZE * (CONSTANTS.common.DEFAULT_PAGE_NO - 1) },
```

Comment:
@Prajwal17Tunerlabs

Reply:
ok, i will use conditions to check for pageSize and pageNo and will use default variables accordingly